topics/statistics/tutorials/CNN/tutorial.md

```diff
@@ -0,0 +1,425 @@
+---
+layout: tutorial_hands_on
+
+title: Introduction to convolutional neural networks (CNN)
+zenodo_link:
+questions:
+- What is a convolutional neural network (CNN)?
+- What are some applications of CNN?
+objectives:
+- Understand the inspiration behind CNN and learn the CNN architecture
+- Learn the convolution operation and its parameters
+- Learn how to create a CNN using Galaxy's deep learning tools
+- Solve an image classification problem on MNIST digit classification dataset using CNN in Galaxy
+requirements:
+  -
+    type: internal
+    topic_name: statistics
+    tutorials:
+      - intro_deep_learning
+time_estimation: 2H
+contributors:
+- kxk302
+
+---
+
+
+
+
+# Introduction
+{:.no_toc}
+
+
+Artificial neural networks are a machine learning discipline that have been successfully applied to problems
+in pattern classification, clustering, regression, association, time series prediction, optimiztion, and control {% cite JainEtAl %}.
+With the increasing popularity of social media in the past decade, image and video processing tasks have become very
+important. The previous neural network architectures (e.g. feedforward neural networks) could not scale up to handle
+image and video processing tasks. This gave way to the development of convolutional neural networks that are specifically
+tailroed to image and video processing tasks. In this tutorial, we explain what convolutional neural networks are, discuss
```

Comment:
`tailroed` should be `tailored`

Reply:
@anuprulez I've made your suggestion into a code suggestion which is easier for contributors to apply :)